src/test/java/org/opensearch/knn/plugin/transport/UpdateModelGraveyardTransportActionTests.java

```diff
@@ -165,4 +188,201 @@ public void testCheckBlock() {
             .getInstance(UpdateModelGraveyardTransportAction.class);
         assertNull(updateModelGraveyardTransportAction.checkBlock(null, null));
     }
+
+    public void testGetIndicesUsingModel() throws IOException, ExecutionException, InterruptedException {
+        // Get update transport action
+        UpdateModelGraveyardTransportAction updateModelGraveyardTransportAction = node().injector()
+            .getInstance(UpdateModelGraveyardTransportAction.class);
+
+        String modelId = "test-model-id";
+        byte[] modelBlob = "testModel".getBytes();
+        int dimension = 2;
+
+        createIndex(MODEL_INDEX_NAME);
+
+        Model model = new Model(
+            new ModelMetadata(
+                KNNEngine.DEFAULT,
+                SpaceType.DEFAULT,
+                dimension,
+                ModelState.CREATED,
+                ZonedDateTime.now(ZoneOffset.UTC).toString(),
+                "",
+                "",
+                "",
+                MethodComponentContext.EMPTY
+            ),
+            modelBlob,
+            modelId
+        );
+
+        // created model and added it to index
+        addDoc(model);
```

Comment:
Is this how we add model in other tests inside this class? I don't think this is a right way to do it.

Reply:
This is how we add models in ModelDaoTests.  I just moved the method from there to the superclass so I could use it in UpdateModelGraveyardTransportActionTests

Reply:
The model isn't created at all in the other tests within this class, they just use a model-id to simulate one existing.